bedrock/products/templatetags/misc.py

```diff
@@ -435,54 +476,50 @@ def relay_monthly_price(ctx, product=RELAY_PRODUCT, plan=RELAY_12_MONTH_PLAN, co
     In Template
     -----------
 
-        {{ relay_monthly_price( product='relay-email',
+        {{ relay_monthly_price_formatted( product='relay-email',
                              country_code=country_code,
                              lang=LANG,
                             ) }}
     """
 
-    available_plans = _relay_get_plans(country_code, lang, product)
-    selected_plan = available_plans.get(plan, RELAY_12_MONTH_PLAN)
-    price = selected_plan.get("price")
-    currency = selected_plan.get("currency")
-    currency_locale = lang.replace("-", "_")
-    amount = _format_currency(price, currency, currency_locale)
+    plan = RELAY_MONTH_PLAN if plan == RELAY_MONTH_PLAN else RELAY_12_MONTH_PLAN
 
-    markup = f"{amount}"
-    return Markup(markup)
+    currency = _relay_get_currency(product, country_code, lang)
+
+    product = product.replace("relay-", "")
+    price = settings.RELAY_STRIPE_PLAN_DATA[product]["prices"][currency][plan]
+
+    price = Decimal(price)
```

Comment:
I'm finding it odd that this is the only place that is converting to `Decimal`. This ultimately gets passed to `babel.numbers.format_currency` and can take a float, Decimal, or string. So for consistency I'd say drop it and just return the price as a float as it is defined in settings.

Reply:
Between a typo and the diff, the purpose of this code was not obvious. 

The decimal is returned by `relay_monthly_price` and is used by `_relay_make_ga_data` where it has some math done on it.

[I added another commit to tidy this up](https://github.com/mozilla/bedrock/pull/13666/commits/8368eeb182ffc14d05f291bd364c893f8015239c) (and fix the typo). It's not big but I hope it makes the purpose of the code more clear.